10up/distributor
# tweak: add new action hooks and deprecated existing

### Description of the Change

As [discussed](https://github.com/10up/distributor/issues/123#issuecomment-1474454938), I deprecated `dt_push_post` and added two action hooks, `dt_push_external_post` and `dt_push_network_post`.


Closes #123

### How to test the Change

- A notice should log in `debug.log` when use `dt_push_post` action hook
-  `dt_push_external_post` and `dt_push_network_post` action hooks should not cause any issue when use

### Changelog Entry

> Added -  `dt_push_external_post` and `dt_push_network_post` action hooks.
> Deprecated - dt_push_post action hook


### Credits

Props @dkotter @peterwilsoncc @dhanendran


### Checklist:
